Formula/pastebinit.rb

```diff
@@ -19,6 +11,7 @@ def install
     inreplace "pastebinit", "/usr/bin/python3", Formula["python3"].opt_bin + "python3"
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
+    rm_r etc/"pastebin.d"
```

Comment:
Usually, formula configurations are left intact during upgrades. What is the reason to update the configuration for all users? Is there is a new configuration option, if so,  you could probably add it formula caveat (wait for a reply from one of the maintainers).